src/Veldrid.ImageSharp/ImageSharpCubemapTexture.cs

```diff
@@ -0,0 +1,103 @@
+﻿using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Advanced;
+using SixLabors.ImageSharp.PixelFormats;
+using System;
+using System.Runtime.InteropServices;
+
+namespace Veldrid.ImageSharp
+{
+    public class ImageSharpCubemapTexture
+    {
+        /// <summary>
+        /// An array of images, each face of a cubemap.
+        /// </summary>
+        public Image<Rgba32>[] CubemapTextures { get; }
+
+        /// <summary>
+        /// The width of a cubemap texture.
+        /// </summary>
+        public uint Width => (uint)CubemapTextures[0].Width;
+
+        /// <summary>
+        /// The height of a cubemap texture.
+        /// </summary>
+        public uint Height => (uint)CubemapTextures[0].Height;
+
+        /// <summary>
+        /// The pixel format cubemap textures.
+        /// </summary>
+        public PixelFormat Format => PixelFormat.R8_G8_B8_A8_UNorm;
+
+        /// <summary>
+        /// The size of each pixel, in bytes.
+        /// </summary>
+        public uint PixelSizeInBytes => sizeof(byte) * 4;
+
+        /// <summary>
+        /// Provides standardized access to the cubemap texture array
+        /// </summary>
+        private const int POSITIVE_X = 0;
+        private const int NEGATIVE_X = 1;
+        private const int POSITIVE_Y = 2;
+        private const int NEGATIVE_Y = 3;
+        private const int POSITIVE_Z = 4;
+        private const int NEGATIVE_Z = 5;
+
+        public ImageSharpCubemapTexture(
+            string negativeZPath,

```

Comment:
We should re-order these so that they are in the standard array-layer order (the order that the constants are in). The stuff in NeoDemo will also need to be re-ordered.